zone/botspellsai.cpp

```diff
@@ -1448,29 +1477,29 @@ Mob* Bot::GetFirstIncomingMobToMez(Bot* caster, int16 spell_id, uint16 spell_typ
 				continue;
 			}
 
+			if (is_pbae_spell) {
```

Comment:
This could probably be simplified down

```cpp
if ((is_pbae_spell ? spell_ae_range : spell_range) < Distance(caster->GetPosition(), npc->GetPosition())) {
    continue;
}
```